pallets/pallet-asset-switch/src/xcm/transact/mock.rs

```diff
@@ -169,9 +178,10 @@ impl ExtBuilder {
 				);
 				Pallet::<MockRuntime>::set_switch_pair_status(switch_pair_info.status).unwrap();
 			}
+			println!("self.1: {:?} ICH BIN HIER DU WIXA", self.1);
```

Comment:
Not sure this belongs here

Reply:
:rofl: ups: [4f612b6](https://github.com/KILTprotocol/kilt-node/pull/686/commits/4f612b65f5c5f9a63105f437de99986efdffe358)